Orange/widgets/visualize/owboxplot.py

```diff
@@ -816,6 +818,15 @@ def draw_axis(self):
         self.box_scene.addLine(
             bottom * scale_x - 4, 0, top * scale_x + 4, 0, self._pen_axis)
 
+    def draw_stat(self):
+        if self.stat_test:
+            l = QGraphicsSimpleTextItem(self.stat_test)
+            l.setFont(self._axis_font)
+            l.setPos((self.scene_min_x + self.scene_max_x)/2 - l.boundingRect().width()/2,
+                     8 + self._axis_font.pixelSize()*2)
+            l.setFlags(l.flags() | QGraphicsItem.ItemIgnoresTransformations)
```

Comment:
You can use `setFlag` instead of `setFlags` (`l.setFlag(QGraphicsItem.ItemIgnoresTransformations)`).